### ISSUE TITLE
React v16 compatibility

### ISSUE THREAD

**Issue by Spy-Seth** [clauderic/react-sortable-hoc#279]
I've try to upgrade to React v16 RC2 my application and I discovers that `react-sortable-hoc@0.6.7` seems to have issue with the next version of React:

```
// node_modules/react-sortable-hoc/dist/commonjs/SortableHandle/index.js:52

Uncaught TypeError: Cannot set property 'sortableHandle' of null
    at _class.componentDidMount (http://localhost:3000/static/js/bundle.js:99380:29)
    at commitLifeCycles (http://localhost:3000/static/js/bundle.js:35983:24)
    at commitAllLifeCycles (http://localhost:3000/static/js/bundle.js:36774:9)
    at HTMLUnknownElement.callCallback (http://localhost:3000/static/js/bundle.js:25751:14)
    at Object.invokeGuardedCallbackDev (http://localhost:3000/static/js/bundle.js:25790:16)
    at invokeGuardedCallback (http://localhost:3000/static/js/bundle.js:25647:27)
    at commitAllWork (http://localhost:3000/static/js/bundle.js:36895:9)
    at workLoop (http://localhost:3000/static/js/bundle.js:37167:13)
    at HTMLUnknownElement.callCallback (http://localhost:3000/static/js/bundle.js:25751:14)
    at Object.invokeGuardedCallbackDev (http://localhost:3000/static/js/bundle.js:25790:16)
    at invokeGuardedCallback (http://localhost:3000/static/js/bundle.js:25647:27)
    at performWork (http://localhost:3000/static/js/bundle.js:37280:7)
    at scheduleUpdateImpl (http://localhost:3000/static/js/bundle.js:37665:19)
    at scheduleUpdate (http://localhost:3000/static/js/bundle.js:37604:12)
    at Object.enqueueSetState (http://localhost:3000/static/js/bundle.js:34127:7)
    at Connect.../node_modules/react-redux/node_modules/react/lib/ReactBaseClasses.js.ReactComponent.setState (http://localhost:3000/static/js/bundle.js:87784:16)
    at Connect.onStateChange (http://localhost:3000/static/js/bundle.js:86468:16)
    at Object.dispatch (http://localhost:3000/static/js/bundle.js:128883:7)
    at dispatch (<anonymous>:2:31875)
    at http://localhost:3000/static/js/bundle.js:128298:7806
    at http://localhost:3000/static/js/bundle.js:128333:60
    at http://localhost:3000/static/js/bundle.js:128371:16
    at dispatch (http://localhost:3000/static/js/bundle.js:128429:18)
    at Object._callee$ (http://localhost:3000/static/js/bundle.js:154573:62)
    at tryCatch (http://localhost:3000/static/js/bundle.js:97611:40)
    at Generator.invoke [as _invoke] (http://localhost:3000/static/js/bundle.js:97845:22)
    at Generator.prototype.(anonymous function) [as next] (http://localhost:3000/static/js/bundle.js:97663:21)
    at step (http://localhost:3000/static/js/bundle.js:154542:191)
    at http://localhost:3000/static/js/bundle.js:154542:361
    at <anonymous>
```

It could be a good idea to prepare the release of a new version that could be compatible with React v16 😄

**Comment by clauderic** [comment]
Fixed in `0.6.8` (#283)

**Comment by Spy-Seth** [comment]
🎉
